Remove automatic linting from build (#83136)
vercel/next.js: docs/01-app/01-getting-started/01-installation.mdx

If you use ESLint, create an explicit config (recommended `eslint.config.mjs`). ESLint supports both [the legacy `.eslintrc.*` and the newer `eslint.config.mjs` formats](https://eslint.org/docs/latest/use/configure/configuration-files#configuring-eslint). See the [ESLint API reference](/docs/app/api-reference/config/eslint#with-core-web-vitals) for a recommended setup.

> > **Good to know**: Prior to Next.js 16, `next build` ran linting automatically. Now, this automatic build-time linting has been removed. Control when linting runs by invoking your linter via npm scripts.

Reviewer: I guess the bot is just wrong here...

> > **Good to know**: Prior to Next.js 16, `next build` ran linting automatically. Now, this automatic build-time linting has been removed. Control when linting runs by invoking your linter via npm scripts.

Reviewer: ```suggestion
> **Good to know**: Starting with Next.js 16, linting no longer runs automatically during `next build`. You now control when linting runs by invoking your linter through npm scripts.
```
